test/gitea_test.exs

```diff
@@ -210,6 +210,15 @@ defmodule GiteaTest do
     teardown_local_and_remote(org_name, repo_name)
   end
 
+  test "commit/3 fails when author is not defined" do
```

Comment:
commit should probably not fail if author is not defined. there is a legit use-case for excluding it. but happy to keep this until someone request different as our _current_ use-case requires it. Thanks.